.golangci.yml

```diff
@@ -68,7 +68,6 @@ linters:
     - gofmt
     - gofumpt
     - goimports
-    - gomnd
```

Comment:
Why this change in this commit?

gomnd was renamed mnd

https://golangci-lint.run/product/changelog/#v1620

Reply:
It was as causing CI to fail and I find mnd to be a lot of noise for not much value so I'm fine without it.

I probably should have removed it in its own PR but I wasn't up for it after adding all the new tests

Reply:
Another PR quickly merged would have been better.

You can still update the PR description to explain your moves

Reply:
updated